javascript/index.js

```diff
@@ -1,65 +1,125 @@
 
 // get the buttons:
-const btnLeft = document.getElementById('btnLeft');
-const btnRight = document.getElementById('btnRight');
+const btnLeft = document.getElementById("btnLeft");
+const btnRight = document.getElementById("btnRight");
 
 // get the DOM elements that will serve us to display the time:
-let minDec = document.getElementById('minDec');
-let minUni = document.getElementById('minUni');
-let secDec = document.getElementById('secDec');
-let secUni = document.getElementById('secUni');
-let milDec = document.getElementById('milDec');
-let milUni = document.getElementById('milUni');
-let splits = document.getElementById('splits');
+let minDec = document.getElementById("minDec");
+let minUni = document.getElementById("minUni");
+let secDec = document.getElementById("secDec");
+let secUni = document.getElementById("secUni");
+let milDec = document.getElementById("milDec");
+let milUni = document.getElementById("milUni");
+let splits = document.getElementById("splits");
 
 function printTime() {
-  // ... your code goes here
+  printMinutes();
+  printSeconds();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  if (chronometer.currentTime < 600) {
+    minDec.innerText = 0;
+    minUni.innerText = chronometer.getMinutes();
+  } else {
+    let time = chronometer.getMinutes().toString();
+    let firstEl = time.charAt(0);
+    let secondEl = time.charAt(1);
+    minDec.innerHTML = Number(firstEl);
+    minUni.innerHTML = Number(secondEl);
+  }
 }
 
 function printSeconds() {
-  // ... your code goes here
+  if (chronometer.currentTime < 10) {
+    secDec.innerText = 0;
+    secUni.innerText = chronometer.getSeconds();
+  } else {
+    let time = chronometer.getSeconds().toString();
+    let firstEl = time.charAt(0);
+    let secondEl = time.charAt(1);
+    secDec.innerHTML = Number(firstEl);
+    secUni.innerHTML = Number(secondEl);
+  }
 }
-
 // ==> BONUS
 function printMilliseconds() {
-  // ... your code goes here
+  if (chronometer.currentTime * 100 < 10) {
+    milDec.innerText = 0;
+    milUni.innerText = chronometer.currentTime;
+  } else if (chronometer.currentTime * 100 <= 99) {
+    let time = chronometer.currentTime.toString();
+    let firstEl = time.charAt(0);
+    let secondEl = time.charAt(1);
+    milDec.innerHTML = Number(firstEl);
+    milUni.innerHTML = Number(secondEl);
+  } else {
+    let time = ((chronometer.currentTime * 100) % 100).toString();
+    let firstEl = time.charAt(0);
+    let secondEl = time.charAt(1);
+    milDec.innerHTML = Number(firstEl);
+    milUni.innerHTML = Number(secondEl);
+  }
 }
 
 function printSplit() {
-  // ... your code goes here
+  const splitLi = document.createElement("li");
+  splitLi.classList.add("splits");
+  splitLi.innerText = chronometer.splitClick();
+  const splitOl = document.getElementById("splits");
+  splitOl.append(splitLi);
 }
 
 function clearSplits() {
-  // ... your code goes here
+  //splitLi.class.remove("splits");
```

Comment:
Os falta poder eliminar los Splits que se hacen.
Para ello lo que tenemos que hacer es seleccionar la clase padre dónde estamos añadiendo todos los splits `id='splits'`, para ello tenemos dos opciones.

1. Utilizar el `querySelector` para seleccionarlo.
2. Usar directamente la variable `splits` (Nota: Al crear un elemento con una **id** lo que ocurre es que se crea una variable global con el mismo nombre que este `id`)

Una vez obtenido el padre podemos utilizar un método `removeChild` [documentación](https://developer.mozilla.org/en-US/docs/Web/API/Node/removeChild)

Si leemos la documentación (leerla 🗡️ ), vemos que tenemos que pasarle a sus hijos cada vez, para poder hace eso tenemos que buscarlos e ir pasandoselos uno por uno. Tenemos muchas formas de hacerlo, una que se me ocurre es: `document.querySelectorAll('.splits')`. Ahora solo falta pasarselo, quedando el código así:

```
function clearSplits() {
  const childrenSplits = document.querySelectorAll('.splits')
  childrenSplits.forEach(child => splits.removeChild(child));
}
```

Nota: No llamáis nunca a esta función. En la linea 120 estáis llamando a un método de `chronometer ` que no existe.